src/components/Components/BuildStatusColumn.tsx

```diff
@@ -21,10 +21,10 @@ import { BuildStatusComponentProps } from '../ComponentsListView/ComponentListVi
 
 export const PR_BOT_NAME = 'appstudio-staging-ci';
 export const GIT_URL_PREFIX = 'https://github.com/';
-export const PAC_ANNOTATION_KEY = 'pipelinesascode';
+export const PAC_ANNOTATION_KEY = 'appstudio.openshift.io/pac-provision';
 
 export const hasPACAnnotation = (component: ComponentKind) =>
-  component.metadata.annotations?.[PAC_ANNOTATION_KEY] === '1';
+  component.metadata.annotations?.[PAC_ANNOTATION_KEY] === 'done';
```

Comment:
There are currently 2 unit test failures. But if I change from `done` to `request` those tests pass.
```suggestion
  component.metadata.annotations?.[PAC_ANNOTATION_KEY] === 'request';
```

Reply:
I think we should keep `done` here, because `request` is transitional state that resolves to `done` or `error`. When the value is `done` this means that PaC provision was successful.

Reply:
Ok that makes sense. I wasn't sure what were the different states.